parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -231,11 +248,15 @@ extractHyperkube() {
 
 installKubeletAndKubectl() {
     if [[ ! -f "/usr/local/bin/kubectl-${KUBERNETES_VERSION}" ]]; then
-        if [[ "$CONTAINER_RUNTIME" == "docker" ]]; then
-            extractHyperkube "docker"
+        if version_gte ${KUBERNETES_VERSION} 1.17; then  # don't use hyperkube
```

Comment:
I think it makes more sense to move this check right after L254

Reply:
Think if >1.17 we should overwrite the current logic of extracting from hyperkube right?
otherwise we will still extract from hyperkube?